db/migrate/20170713183443_create_log_events.rb

```diff
@@ -0,0 +1,14 @@
+class CreateLogEvents < ActiveRecord::Migration
+  def change
+    create_table :log_events do |t|
+      t.references :loggable, polymorphic: true
```

Comment:
Could you include the index with `index: true` here? I thought that it worked for polymorphic references as well.